### DISCUSSION TITLE
Great concept

### DISCUSSION THREAD

**Discussion by ALFAIthildur** [Truinto/SpellPouch#6] [General]
Just wanted to chime in to say the mod idea is a terrific concept. I've downloaded it but haven't had the courage to install it yet looking at the warning that saves can become dependent on the mod, and also the lack of feedback/input page which gives me no indication whether people are having any issues or not. I generally wait until a new mod that's going to lock up saves becomes established, i.e. 'ripens' a bit with visible feedback from people who've experienced good results with it. Hopefully a comments section does go up at some point over on Nexus Mods for this project.

**Comment by Truinto** [comment]
This mod is in use for some time now from various people. And I don't plan to change the mod. Also the effects on your saves are minimal. Solely the custom slots in your action bar are saved. That can be undone by removing all of the slots from your action bars (although this includes all companions, some of which might not be in your party at all times).
Tho I think you would be able to load the save even without clearing your action bar. I tried that once without issue. And as a bonus it's possible to use BlueprintPurge (with the right settings) to clear these entries as well.

So yeah, the warning is overly cautious. I just don't want people to flame me.

I opened the comment section, as I can understand that people wouldn't want a Github account just to comment here. But I am not fond of Nexus, so I probably won't see any of them.

**Reply by ALFAIthildur** [reply]
Looking through Discord comments, it looks like someone has commented that Spellpouch is not working with the updated versions of the game and/or there are issues with empty pouches - is this something others have reported?

"Currently no (Spellpouch does not work with 1.4). On the surface it works but it doesn't save the settings, leaving you with a bunch of empty pouches. It's currently  my biggest holdout. Should be a feature. Too many skills"

**Reply by RonaldKendall** [reply]
After about a day of trying to isolate the problem, I found that I was having issues with empty pouches due to a conflict in Tabletop Tweaks (Core).  Which sucks because I like TTT(Base) which is dependent on (Core).  Disabling TTT(Core) gave me my elemental blasts back. I verified that I could group spells on the toolbar, but not the spell tab, like you can on the ability tab.

ALFA, there is a feature in the ToyBox mod on the Bag of Tricks tab in the Quality of Life section that makes spell, ability (and item) tabs wider as needed, preventing vertical overflow offscreen.  If you can't do without TTT, you can check that box and be spared the pain of not being able to access so many abilities or spells (like 9 level 1 classes with cantrips and orisons preventing you from being able to select spell level to view other spells).
